integration/src/test/java/edu/unc/lib/boxc/integration/web/access/SearchEndpointIT.java

```diff
@@ -295,4 +295,51 @@ public void testSearchPublicUserWithMetadataOnlyPermissionCanSeeMetadata() throw
             assertEquals(6, metadata.size());
         }
     }
+
+    @Test
+    public void testSearchWithCreatedDateRangeSpecified() throws Exception {
+        createDefaultObjects();
+        var adminUnit = adminUnitFactory.createAdminUnit(
```

Comment:
Since all 3 of these tests depend on having the same 2 objects created, it might make sense to either create them in a private method (could return the objects in a list or array), or to combine the 3 tests so the setup only happens once. The latter is the route I ended up taking over here:
https://github.com/UNC-Libraries/box-c/pull/1443/files#diff-e3d038bc6b26620f1703cd71bf02a84da5b4ba6f5779abca61ad7edc54647b39R301

Reply:
cool, i've made a private method. i only see two tests though so have only added it to the two. if i've missed one let me know!